src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1229,4 +1229,41 @@ private AdsDataProvider() {
         }
     };
   }
+
+  @DataProvider
+  public static Object[][] adsUapMercury() {
+    return new Object[][]{
+        {
+            new Page("project43", "SyntheticTests/UAP"),
+            Arrays.asList(
+                ImmutableMap.<String, Object>builder()
+                    .put("slotName", AdsContent.MOBILETOP_LB)
+                    .put("slotSize", new Dimension(360, 540))
+                    .put("lineItemId", 297978612)
+                    .put("src", "mobile")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "mobile" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)